src/main/resources/io/jenkins/plugins/orka/AgentTemplate/config.jelly

```diff
@@ -18,6 +18,10 @@
         <f:entry title="${%# of CPUs}" field="numCPUs">
           <c:select />
         </f:entry>
+
+        <f:entry title="${%Memory in GiB}" field="memory">
+          <f:number />
```

Comment:
I think this should be string.
So we can allow:

- "auto" as a value (which would be the default)
- Integers

Why?
Because seeing 0 wil make everyone change it. And people will not know how to get auto value.

Also, we should introduce a nice tooltip message explaining what this is

Reply:
![image](https://user-images.githubusercontent.com/35697323/166553025-c5f74ca2-de43-431d-b5ad-0377223e2410.png)